Inflattices (#1319)
UniMath/agda-unimath: src/order-theory/greatest-lower-bounds-large-posets.lagda.md

    is-binary-lower-bound-Large-Poset P a b y ↔ leq-Large-Poset P y x
```

> ### The predicate on large posets of being a greatest lower bound of a family of elements

Reviewer: ```suggestion
### The predicate of being a greatest lower bound of a family of elements in a large poset
```

        ↔ leq-Π-Large-Poset P u z
          by iff-Π-iff-family (λ i → H i (u i))
```

> ### A family of greatest lower bounds of families of elements in a family of large poset is a greatest lower bound in their dependent product

Reviewer: ```suggestion
### Pointwise greatest lower bounds are greatest lower bounds in the dependent product
```
